compiler/rustc_session/src/options.rs

```diff
@@ -1832,7 +1838,7 @@ written to standard error output)"),
     trap_unreachable: Option<bool> = (None, parse_opt_bool, [TRACKED],
         "generate trap instructions for unreachable intrinsics (default: use target setting, usually yes)"),
     treat_err_as_bug: Option<NonZeroUsize> = (None, parse_treat_err_as_bug, [TRACKED],
-        "treat error number `val` that occurs as bug"),
+        "treat the nth error that occurs as bug (default: 0 - don't treat errors as bugs)"),
```

Comment:
```suggestion
        "treat the `val`th error that occurs as bug (default `val` is 1 if this option is specified, and errors are not treated as bugs if this option is not specified)"),
```
`-Z help` outputs `-Z                      treat-err-as-bug=val -- *description*`, that's why `val` was used here.

Also, the wording is confusing - the default is also `1`, because `-Ztreat-err-as-bug` means `-Ztreat-err-as-bug=1`.